tests/support/loadJsdom.ts

```diff
@@ -29,6 +29,12 @@ global.requestAnimationFrame = (cb: (...args: any[]) => {}) => {
 	return true;
 };
 
+global.requestIdleCallback = (cb: (...args: any[]) => {}) => {
```

Comment:
reminder that we would want to address this in https://github.com/dojo/test-extras/blob/master/src/support/loadJsdom.ts as well

Reply:
Yup! Thanks!